apps/emqx_authn/src/emqx_authn.erl

```diff
@@ -126,8 +126,8 @@ create_chain(#{id := ID}) ->
                                    created_at = erlang:system_time(millisecond)},
                     mnesia:write(?CHAIN_TAB, Chain, write),
                     {ok, serialize_chain(Chain)};
-                [_ | _] ->
-                    {error, {already_exists, {chain, ID}}}
+                [Chain] ->
+                    {ok, serialize_chain(Chain)}
```

Comment:
I am not sure about this part. Looking at the code, it appears that this code should always fail in the clustered setup, but for some reason it only fails on the replicant nodes. Perhaps I missed some part.